ReleaseProcessScript/tools/Core/semver_functions.ps1

```diff
@@ -85,44 +85,48 @@ function Get-Possible-Versions-Hotfix ($Version, $MeantForCurrentVersion = $fals
 {
   $Match = Parse-Semver $Version
 
-  $Major = $Match.Groups["major"].ToString()
-  $Minor = $Match.Groups["minor"].ToString()
-  $Patch = $Match.Groups["patch"].ToString()
+  $OriginalMajor = $Match.Groups["major"].ToString()
+  $OriginalMinor = $Match.Groups["minor"].ToString()
+  $OriginalPatch = $Match.Groups["patch"].ToString()
 
   if (-not $MeantForCurrentVersion)
   {
-    $Patch = [string](1 + $Patch)
+    $NextPatch = [string](1 + $OriginalPatch)
   }
-  $NextPossiblePatch = "$($Major).$($Minor).$($Patch)"
+  else 
+  {
+    $NextPatch = $OriginalPatch
+  }
+  $NextPossiblePatchVersion = "$($OriginalMajor).$($OriginalMinor).$($NextPatch)"
 
   #Compute 1.2.3-alpha.4
   if ($Match.Groups["pre"].Success)
   {
-    $Pre = $Match.Groups["pre"].ToString()
-    $PreVersion = $Match.Groups["preversion"].ToString()
-    $NextPossibleFullVersion = "$($Major).$($Minor).$($Patch)"  
+    $PreReleaseType = $Match.Groups["pre"].ToString()
+    $OriginalPreReleaseNumber = $Match.Groups["preversion"].ToString()
+    $NextPossibleFullVersion = "$($OriginalMajor).$($OriginalMinor).$($NextPatch)"  
 
-    $NextPreVersion = [string](1 + $PreVersion)
-    $NextPossiblePreVersion = "$($Major).$($Minor).$($Patch)-$($Pre).$($NextPreVersion)" 
+    $NextPreReleaseNumber = [string](1 + $OriginalPreReleaseNumber)
+    $NextPossiblePreReleaseVersion = "$($OriginalMajor).$($OriginalMinor).$($OriginalPatch)-$($PreReleaseType).$($NextPreReleaseNumber)" 
 
-    if ($Pre -eq "alpha")
+    if ($PreReleaseType -eq "alpha")
     {
-      $NextPossiblePre = "$($Major).$($Minor).$($Patch)-beta.1"
+      $NextPossibleBetaVersion = "$($OriginalMajor).$($OriginalMinor).$($OriginalPatch)-beta.1"
```

Comment:
I would prefer `Current...` over `Original...`. Other than that, it seems fine.

Reply:
Ok, I'm just waiting for @MichaelKetting's general opinion before renaming them so that I don't have to change the PR too often.